Platform/AlderlakeBoardPkg/BoardConfigAzb.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, *args, **kwargs):
         self._EXTRA_INC_PATH                  = ['Silicon/AlderlakePkg/Azb/Include']
         self._FSP_PATH_NAME                   = 'Silicon/AlderlakePkg/Azb/FspBin'
         self.MICROCODE_INF_FILE               = 'Silicon/AlderlakePkg/Microcode/MicrocodeAzb.inf'
+        self.FSP_INF_FILE                     = 'Silicon/AlderlakePkg/FspBin/FspBinAzb.inf'
```

Comment:
The name "Azb" doesn't match with the one in actual file name "azb". maybe need update the actual one to avoid build failure.

Reply:
Strange, my local file name file name case is correct, will move to Linux to check.